internal/pkg/cmd/prerunner.go

```diff
@@ -149,7 +149,7 @@ func (c *AuthenticatedCLICommand) GetFlinkGatewayClient() (*ccloudv2.FlinkGatewa
 			return nil, err
 		}
 
-		authToken, err := pauth.GetJwtTokenForV2Client(ctx.GetState(), ctx.GetPlatformServer())
+		authToken, err := pauth.GetDataplaneToken(ctx.GetState(), ctx.GetPlatformServer(), map[string]any{})
```

Comment:
Could we replace `map[string[]any{}` with `nil`?

Reply:
If it gets encoded to `{}` in JSON, then sure. Let me investigate!

Reply:
Just checked and it doesn't work, we need to use an empty map

Reply:
Well, that's unfortunate.